docs/src/main/paradox/docs/delta/api/resources-api.md

```diff
@@ -34,6 +34,28 @@ The json payload for create and update operations cannot contain keys beginning
 
 @@@
 
+## Nexus metadata
+
+When using the endpoints described on this page, the responses can contain the following metadata:
+
+- `_self`: unique address of this resource in the current Nexus Delta deployment
+    - Each resource has an `@id`, which is only unique within a project. The `_self` in contrast to the `@id` will
+      uniquely identify a resource across all projects in Nexus Delta.
+    - A `GET` request can be performed on a `_self` to obtain the underlying resource
```

Comment:
Maybe something like "because _self is the Nexus address of the resource, a GET request can be performed at the value in order to obtain the underlying resource"?

Reply:
Rewritten